tests/pfc/files/configs/pfc.py

```diff
@@ -0,0 +1,399 @@
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.reboot import logger
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.ixia.ixia_helpers import IxiaFanoutManager, get_location
+
+from tests.common.ixia.common_helpers import get_vlan_subnet, \
+    get_addrs_in_subnet
+
+###############################################################################
+# Imports for Tgen and IxNetwork abstract class
+###############################################################################
+
+from abstract_open_traffic_generator.port import Port
+from abstract_open_traffic_generator.config import Options
+from abstract_open_traffic_generator.config import Config
+
+from abstract_open_traffic_generator.layer1 import\
+    Layer1, OneHundredGbe, FlowControl, Ieee8021qbb
+
+from abstract_open_traffic_generator.device import\
+     Device, Ethernet, Ipv4, Pattern
+
+from abstract_open_traffic_generator.flow import\
+    DeviceTxRx, TxRx, Flow, Header, Size, Rate,\
+    Duration, FixedSeconds, PortTxRx, PfcPause, EthernetPause, Continuous
+
+from abstract_open_traffic_generator.flow_ipv4 import\
+    Priority, Dscp
+
+from abstract_open_traffic_generator.flow import Pattern as FieldPattern
+from abstract_open_traffic_generator.flow import Ipv4 as Ipv4Header
+from abstract_open_traffic_generator.flow import Ethernet as EthernetHeader
+from abstract_open_traffic_generator.port import Options as PortOptions
+
+def calculate_priority_vector(v) :
+    s = 0
+    for i in range(8)  :
+        if v[i] != '0' :
+           s += 2**(7 - i)
+           print(i)
```

Comment:
please remove this debug statement. If needed, use log instead.

Reply:
I have removed the print statement.